python-main.js

```diff
@@ -356,6 +356,15 @@ function web_editor(config) {
         } else {
             return alert(config.translate.alerts.empty);
         }
+        
+        if (micropythonFs.getStorageRemaining() < 0){
+            alert(config.translate.alerts.out_of_space);
+            if (isModule){
+                micropythonFs.remove(filename);
+                return alert(config.translate.alerts.module_out_of_space);
```

Comment:
So, if a module is too large the user will be prevented with two alerts one after the other.
Could we modify this to ensure that only one alert is presented?

Reply:
Actually since it's okay not to show the warning when the file is not a module (as discussed in the issues, they will know when downloading the hex file), then we can put this check and alert inside the `if (isModule) {` statement in line 368.